errors.go

```diff
@@ -104,3 +116,33 @@ type InvalidKeyFileError struct {
 func (e InvalidKeyFileError) Error() string {
 	return fmt.Sprintf("invalid key data file: %s", e.msg)
 }
+
+func isInvalidKeyFileError(err error) bool {
+	var e InvalidKeyFileError
+	return xerrors.As(err, &e)
+}
+
+// LockAccessToSealedKeysError is returned from ActivateVolumeWithTPMSealedKey if an error occurred whilst trying to lock access
+// to sealed keys created by this package.
+type LockAccessToSealedKeysError string
+
+func (e LockAccessToSealedKeysError) Error() string {
+	return "cannot lock access to sealed keys: " + string(e)
+}
+
+// ActivateWithTPMSealedKeyError is returned from ActivateVolumeWithTPMSealedKey if activation with the TPM protected key failed.
+type ActivateWithTPMSealedKeyError struct {
+	// TPMErr details the error that occurred during activation with the TPM sealed key.
+	TPMErr error
+
+	// RecoveryErr details the error that occurred during activation with the fallback recovery key, if activation with the recovery key
+	// was also unsuccessful.
+	RecoveryErr error
```

Comment:
this should be RecoveryKeyErr or RecoveryKeyRecourseErr, see other discussion about just using plain "Recovery"

Reply:
I suppose actually, RecoveryKeyUsageErr given the last naming decision